client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.routes.js

```diff
@@ -0,0 +1,23 @@
+angular.module('App').config(($stateProvider) => {
+  $stateProvider.state('app.dedicatedClouds.datacenter.drp', {
+    url: '/drp',
+    views: {
+      'pccDatacenterView@app.dedicatedClouds.datacenter': {
+        component: 'dedicatedCloudDatacenterDrp',
+      },
+    },
+    translations: ['../../../ip'],
+    params: {
+      selectedDrpType: null,
+    },
+    resolve: {
+      datacenterHosts: ($stateParams, DedicatedCloud) => DedicatedCloud
```

Comment:
```suggestion
      datacenterHosts: /* @ngInject */ ($stateParams, DedicatedCloud) => DedicatedCloud
```

It's annoying but we decided to right them explicitly even though they are not mandatory

Reply:
Yes I forgot these ones :)